### PR TITLE
Don't Require Running in Release to Test Mood

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**

### CODE REVIEW

**Comment by DEATHB4DEFEAT** [comment]
wtf conflicts?? I made sure to be up to date with master

File: Content.Shared/CCVar/CCVars.cs [hunk 1]
```diff
@@ -2567,7 +2567,11 @@ public static readonly CVarDef<float>
         #region Mood System
 
         public static readonly CVarDef<bool> MoodEnabled =
+        #if RELEASE
             CVarDef.Create("mood.enabled", true, CVar.SERVER);
+        #else
+            CVarDef.Create("mood.enabled", false, CVar.SERVER);
+        #endif
```
Comment (VMSolidus) [inline Content.Shared/CCVar/CCVars.cs:2574]: And now Mood doesn't work in Release configuration.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

File: Content.Shared/CCVar/CCVars.cs (hunk 1 above)
Comment (DEATHB4DEFEAT) [reply Content.Shared/CCVar/CCVars.cs:2574]: whar

Comment (VMSolidus) [reply Content.Shared/CCVar/CCVars.cs:2574]: In all instances of (!_config.GetCVar(CCVars.MoodEnabled)

Comment (DEATHB4DEFEAT) [reply Content.Shared/CCVar/CCVars.cs:2574]: why?

**Review by VMSolidus — APPROVED** [review]

**Review by sleepyyapril — APPROVED** [review]

**Pull request merged**